x/datapool/types/pool.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/v044_temp/address"
```

Comment:
How about we refactor this package to under `x/` directory?

Reply:
I don't have any idea for this. It is not a big deal I think, so we can move it. Could you comment your thought @gyuguen and @youngjoon-lee ?

Reply:
My guess is that `v044_temp` is not a module, so it shouldn't go under `x/`.
Ummm.... If I had to move this, it would be to move it to the root folder.